indico/web/client/js/react/components/pictures/PictureManager.jsx

```diff
@@ -201,7 +201,7 @@ const PictureManager = ({
       fetch(imageSrc)
         .then(res => res.blob())
         .then(blob => {
-          const file = new File([blob], 'picture.png', blob);
+          const file = new File([blob], 'picture.jpeg', blob);
```

Comment:
```suggestion
          const file = new File([blob], 'picture.jpg', blob);
```

`.jpeg` is valid but rather uncommon.